dspace/modules/dryad-widgets/bin/travis-install-maven.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
I probably added it when I had a separate repo (instead of a dspace/module) for dryad-widgets, and forgot to remove it. I'll remove it from the dryad-widgets module